index.html

```diff
@@ -0,0 +1,293 @@
+      identify credential via {{CredentialsContainer/requestIdentity()}}.
+    </p>
+    <p>
+      The {{IdentityRequestProvider/protocol}} member's value is be one of the
+      keys defined in [[[#protocol-registry]]].
+    </p>
+    <h3>
+      The `request` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="IdentityRequestProvider">request</dfn> member is
+      the request to be sent to the [=identity credential provider=].
+    </p>
+    <h3>
+      The `publicKey` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="IdentityRequestProvider">publicKey</dfn> member is
+      the public key of the [=identity credential provider=].
+    </p>
+    <h2>
+      Requesting a digital identity credential
+    </h2>
+    <p>
+      The following steps are taken to <dfn>request digital identity
+      credentials</dfn>, with {{IdentityRequestOptions}} |options|:
+    </p>
+    <ol>
+      <li>Let |promise| be [=a new promise=].
+      </li>
+      <li>Return |promise| and continue running the following steps [=in
+      parallel=].
+      </li>
+      <li>For each |provider:IdentityRequestProvider| of |options|'s
+      {{IdentityRequestOptions/providers}}.
+        <ol>
+          <li>[=verify the public key=] of |provider|.
+          </li>
+        </ol>
+      </li>
+      <li>...present [=identity credential provider=] to user.
+      </li>
+    </ol>
+    <h2>
+      The `DigitalCredential` interface
+    </h2>
+    <p>
+      The {{DigitalCredential}} interface represents a [=digital identity
+      credential=].
+    </p>
+    <pre class="idl">
+    [Exposed=Window, SecureContext]
+    interface DigitalCredential : Credential {
+      readonly attribute DOMString protocol;
+      readonly attribute ArrayBuffer data;
+    };
+    </pre>
+    <h2>
+      Public key verification
+    </h2>
+    <p>
+      The following steps are taken to <dfn>verify the public key</dfn> of a
+      [=identity credential provider=], with {{CryptoKey}} |key|:
+    </p>
+    <ol>
+      <li>...restrictions on type, format, and usage of |key|...
+      </li>
+    </ol>
+    <h2 id="protocol-registry">
+      Identity Request Protocols Registry
+    </h2>
+    <p>
+      The following is the initial registry of [=digital credential/request
+      protocols=] that are supported by this specification.
+    </p>
+    <p>
+      User agents MUST support the following [=digital credential/request
+      protocols=]:
+    </p>
+    <table>
+      <thead>
+        <tr>
+          <th>
+            Protocol identifier
+          </th>
+          <th>
+            specification
+          </th>
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            `"mdoc"`
+          </td>
+          <td>
+            [[[ISO-18013-5]]]
+          </td>
+          <td>
+            A mobile driving license (mDL) request.
```

Comment:
does this work for other document types, or just mobile drivers licenses?

Reply:
I was just starting with this because it will be a miracle if we even get driver licenses.

Reply:
Note: removing this and moving discussion to https://github.com/WICG/digital-credentials/issues/58